README.md

```diff
@@ -39,6 +39,8 @@ your will.
 There is no set purpose.  What ChaosBot makes itself into is entirely up to
 the imagination of the open source community.
 
+Wondering how to contribute? Try adding a feature requested [here](https://github.com/chaosbot/chaos/wiki/Feature-Requests).
```

Comment:
Then why not "Try adding a feature request", since "a feature request" is the thing that can be added, not "a feature requested"?